tests/end2end/features/test_misc_bdd.py

```diff
@@ -29,6 +29,7 @@
 from qutebrowser.utils import docutils
 from qutebrowser.browser import pdfjs
 
+from end2end.features.test_scroll_bdd import check_scrolled, check_not_scrolled
```

Comment:
Okay, my bad, that doesn't work... What you can do is moving those functions from `test_scroll_bdd.py` to `tests/end2end/features/conftest.py` - then they'll be available in any `.feature` file.